ui/src/pages/backoffice/ILL/LibraryEditor/components/LibraryForm/components/AlternativeAbstracts.js

```diff
@@ -0,0 +1,45 @@
+import React, { Component } from 'react';
```

Comment:
What are all the components e.g. `LibraryForm/components/AlternativeAbstracts.js` and similar for? Maybe a copy/paste from the Document not needed?

Reply:
Oups :P